app/controllers/kaui/invoices_controller.rb

```diff
@@ -39,7 +65,7 @@ def pagination
         end
         formatter = lambda do |invoice|
           row = [view_context.link_to(invoice.invoice_number, view_context.url_for(controller: :invoices, action: :show, account_id: invoice.account_id, id: invoice.invoice_id))]
-          row += Kaui.invoice_search_columns.call(invoice, view_context, cached_options_for_klient)[1]
```

Comment:
`invoice_search_columns` being a public API (this is how customers can customize the columns), let's keep the last argument (not used by default, but might be used by existing customizations).